src/controllers/Definition/DefinitionGetController.ts

```diff
@@ -0,0 +1,51 @@
+import db from '../../database';
+import * as TermSelectDAO from '../../dao/Term/TermSelectDAO';
+import * as DefinitionSelectDAO from '../../dao/Definition/DefinitionSelectDAO';
+import * as PosSelectDAO from '../../dao/Pos/PosSelectDAO';
+import * as UsageSelectDAO from '../../dao/Usage/UsageSelectDAO';
+import * as OriginSelectDAO from '../../dao/Origin/OriginSelectDAO';
+import MarmoymError from '../../models/MarmoymError';
+import ErrorType from '../../models/ErrorType';
+import { transaction } from '../../database/databaseUtils';
+
+export async function getDefinitionByTermId(req) {
+  const termSelected = await TermSelectDAO.selectTermByTermId(req.termId);
+
+  if (termSelected.length == 0) {
+    throw new MarmoymError(ErrorType.Term.TERM_NOT_FOUND);
+  } else {
+    termSelected[0]['defs'] = [];
+    //TODO 현재는 최신순으로 등록된 defs를 가져오지만 추후 vote순으로 가져오기
+    const defs = await DefinitionSelectDAO.selectRecentlyCreatedDefinitionsByTermId(req.termId, req.offset, 5);
+
+    await Promise.all(defs.map(async defObj => {
+      //get Pos 
+      const posSelected = await PosSelectDAO.selectPosByDefinitionId(defObj.id);
+      defObj['poss'] = [];
+      await Promise.all(posSelected.map(posObj => {
```

Comment:
Why posObj? Can it be just pos?

same as in def, usage, origin ,,.

Reply:
My style

Reply:
If you insist, I wouldn't complain. But know that putting a type in the variable name is discouraged. We habve TypeScript, in that case.